pkg/analytics/track.go

```diff
@@ -371,6 +372,14 @@ func TrackBuildPullError(oktetoBuilkitURL string, success bool) {
 	track(buildPullErrorEvent, success, props)
 }
 
+// TrackContextDelete sends a tracking event to mixpanel indicating one or more context have been deleted
+func TrackContextDelete(ctxs int) {
+	props := map[string]interface{}{
+		"hasMoreThanOneContext": ctxs,
```

Comment:
we need to add a proper name to the namber of context deleted as suggested in https://github.com/okteto/okteto/pull/3886#discussion_r1288170817. `totalContextsDeleted` would be a good name 👍